openthread/CMakeLists.txt

```
@@ -6,6 +6,19 @@
 include(${NRFXLIB_DIR}/common.cmake)
 nrfxlib_calculate_lib_path(lib_path)
 
+file (READ "../../modules/lib/openthread/include/openthread/instance.h" INSTANCE_FILE_CONTENT)
```

Please refer to the OpenThread module dir.
```suggestion
file(READ "${ZEPHYR_OPENTHREAD_MODULE_DIR}/../include/openthread/instance.h" INSTANCE_FILE_CONTENT)
```